packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx

```diff
@@ -169,8 +169,6 @@ export const RichTextBlock = memo(function RichTextBlock({
           setLocation(editor.selection)
         }}
         onKeyDown={e => {
```

Comment:
Fixed by upgrading @react/types to ^17.0.14, as <root> and <root>/packages/editor had different versions but were both using the same interface.